src/util/getStripeClient.ts

```diff
@@ -13,14 +13,13 @@ export function getStripeClient({
   const stripeClient = new Stripe(apiKey, {
     apiVersion,
     typescript,
+    appInfo: {
```

Comment:
refs: https://github.com/stripe/stripe-node/wiki/Migration-guide-for-v8#add-appinfo-to-config-object